site/_docs/history.md

```diff
@@ -29,17 +29,93 @@ Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
 <!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.36.0">1.36.0</a> / XXXX-XX-XX
-{: #v1-36-0}
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.37.0">1.37.0</a> / XXXX-XX-XX
+{: #v1-37-0}
 
-This release comes x months after [1.35.0](#v1-35-0),
+This release comes x months after [1.36.0](#v1-36-0),
 contains contributions from x contributors, and resolves x issues.
 
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-37-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 21.0 to 32.1.3-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-37-0}
+
+#### Dependency version upgrade
+{: #dependency-1-37-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-37-0}
+
+#### Build and test suite
+{: #build-1-37-0}
+
+#### Web site and documentation
+{: #site-1-37-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.36.0">1.36.0</a> / 2023-11-XX
+{: #v1-36-0}
+
+This release comes 3 months after [1.35.0](#v1-35-0),
+contains contributions from 30 contributors, and resolves 125 issues.
+
+Among other new features, it's worth highlighting the adding of 30 new SQL functions in various libraries such as BigQuery and Spark, many improvements hardening `TABLESAMPLE` , and also the following features:
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5615">CALCITE-5615</a>]
+ Run SQL Logic Test suite using Calcite's HSQLDB JDBC adapter
```

Comment:
This is not a new feature but a test improvement. Not sure it fits here.

Reply:
I have the same feeling when I added it here, I thought it is a major work that I wanted to highlight it. How about adding a line below 'highlighting new features' paragraph like:

> In addition new features, it's also worth highlighting the integrating of <a href="https://issues.apache.org/jira/browse/CALCITE-5615">SQL Logic Test suite</a>.

Reply:
+1 It's a major improvement to the test suite and therefore qualifies as a feature.